recipes/Kili_fine_tuning_tutorial.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    sample_text2 = "Most Of Beijing To Be Tested For COVID-19 Amid Lockdown Worry. While only 70 cases have been found since the outbreak surfaced, authorities have followed a “zero-COVID” approach to try to prevent a further spread of the virus."
</pre><p>remove this line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>Commented out. This is a sample line in case if someone wanted to experiment a bit more.</p>